packages/smart-contracts/README.md

```diff
@@ -127,6 +135,59 @@ Environment variables needed: `ADMIN_PRIVATE_KEY`
 yarn hardhat prepare-live-payments --network private
 ```
 
+### Deployment through request deployer
+
+The request deployer enables multichain deployment of several smart contract at predefined address.
+For now it is used to deploy:
+
+- `EthereumProxy`
+- `EthereumFeeProxy`
+
+Next versions of our contracts will also be deployed via the request deployer
+
+#### Deploy the request deployer
+
+Environment variables needed: `DEPLOYER_MASTER_KEY`
+
+```bash
+yarn hardhat deploy-deployer-contract --network <NETWORK>
+```
+
+The resulting contract should be deployed at `0xE99Ab70a5FAE59551544FA326fA048f7B95A24B2` (live chains)
+
+#### Compute the contract addresses
+
+Run:
+
+```bash
+yarn hardhat compute-contract-addresses
+```
+
+It will compute the addresses of the contract to be deployed via the request deployer.
+
+#### Deploy the contracts
```

Comment:
In this section, you can detail a bit further the outcome of the command (if already deployed, shows the address and skips...)